### ISSUE TITLE
[bug] Bot analytics > Engagement > Guilds graph reverst to 0 without if nothing happens that day

### ISSUE THREAD

**Issue by CTNOriginals** [discordlist-gg/beta-issues#334]
### Page

https://discordlist.gg/bot/1308469474768457748/manage/analytics

### Description

On the analytics guilds page for a bot, the graph goes down to the bottom line if nothing changes that day, if the api does not receive a callback the entire day the graph just shows a massive drop as if it were to 0 guilds for that day.
![image](https://github.com/user-attachments/assets/e554c5e3-170f-429a-abda-050bc297f5a9)

### Notify on issue updates

- [X] Yes

### Discord username

ctn

**Comment by audn** [comment]
This isn’t actually a bug, but I agree it shouldn’t treat a missed day as '0'. I’ll look into a better solution – thanks!

**Comment by CTNOriginals** [comment]
Yeah i also realized that it might just be a bar graph like the other graphs but drawn with a line instead as it functions like that if you look at the 24 hour time frame.
![image](https://github.com/user-attachments/assets/5f575846-2710-4709-a0a0-1147ac8a5bee)
But yeah, thats pretty confusing and i would also much rather like to see the steady growth of the guilds the bot is in drawn as a line.

Good luck on the process though!
